ext/FHistMakieExt.jl

```diff
@@ -6,6 +6,39 @@ isdefined(Base, :get_extension) ? (using Makie) : (using ..Makie)
 
 import FHist: stackedhist, stackedhist!
 
+const SAFE_LOG = Ref(true)
+
+"""
+    set_safe_log(x::Bool)
+
+Turn on/off SAFE_LOG, to plot histograms without/with negative values. By default is `true`.
+"""
+function set_safe_log(x::Bool)
```

Comment:
I don't think we can add new thinsg into the namespace via Pkg Extension mechanism.

I will see if we can resolve this using plotting arguments

Reply:
My bad, I just tested that the clipping was doing the right thing, but haven't check that the switch was actually working...

Reply:
yeah no worries, this is more than enough! I will try to make it nicer to change for users!